modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1023,6 +1023,11 @@
     <string name="settings_lock_image">lock image</string>
     <string name="settings_manage_downloads_include_starred">Include starred</string>
     <string name="settings_manage_downloads_total">Total</string>
+    <string name="settings_media_actions_prioritize_title">Prioritize your custom media actions</string>
+    <string name="settings_media_actions_prioritize_subtitle">Your top actions will be available.</string>
+    <string name="settings_media_actions_customise">Customise media actions</string>
+    <string name="settings_media_actions_hide_title">Show custom media actions</string>
+    <string name="settings_media_actions_hide_subtitle">Show custom media actions in Android 13 media notifications and Android Auto.</string>
```

Comment:
Want to also update these string names from "hide" to "show"?

Reply:
I went ahead and added this change in [753d545](https://github.com/Automattic/pocket-casts-android/pull/719/commits/753d5451ce0e5f55e751f1bd5ae6d82e51ae81fb).